gen/genoutput/libhsa_api.jl

```diff
@@ -0,0 +1,1026 @@
+
```

Comment:
Looks like this is duplicated into `src/HSA/libhsa_api.jl` (same for the other Julia wrappers); can we keep just one copy around of each?

Reply:
sure